### DISCUSSION TITLE
Algolia + GitHub Action

### DISCUSSION THREAD

**Discussion by dieghernan** [mmistakes/minimal-mistakes#2807] [Show and tell]
Hi,

Since Travis CI has changed their billing, some users relying on Travis for building their Algolia indexes may be facing some issues.

I just want to share with you an alternative using `jekyll-algolia` + GH Actions (already shared on https://github.com/algolia/jekyll-algolia/issues/165 and accepted by `jekyll-algolia` maintainers here https://github.com/algolia/jekyll-algolia/commit/12437512c1974675f5aa8399d3ea44c759d980a4 , although it is not available on the web docs yet).

Based on that I included in the GH Marketplace an action https://github.com/marketplace/actions/algolia-jekyll-action that I consider it can be interesting for the users of minimalmistakes. It is fast and allows you to get rid of third party providers.

See [here](https://github.com/dieghernan/chulapa/actions?query=workflow%3Aalgolia-search) a live implementation.

Regards

**Comment by mmistakes** [comment]
Cool. If you have other things like this to share the [Discussions tab](https://github.com/mmistakes/minimal-mistakes/discussions) is a good place. There's a category marked "Show and Tell".
Previously GH allowed me to migrate issues to discussions, but that's been turned off temporarily. Once enabled I'll move this over.

**Reply by xjlin0** [reply]
Just a follow up, as of today, by direct copying&pasting the settings from [algolia-jekyll-action page](https://github.com/marketplace/actions/algolia-jekyll-action), the algolia auto-indexing works upon push to personal github page.  I also need to add more nodes to index like `nodes_to_index: 'p,blockquote,li,div,em,strong,sup,a,table'`.
